src/Main.hs

```diff
@@ -4,4 +4,4 @@ import Boids
 window = InWindow "Boids" (800, 600) (10, 10)
 
 main = do boids <- initialState
-          simulate window white 30 boids renderState nextState
+          play window white 30 boids renderState handleInput (nextState undefined)
```

Comment:
Get rid of that `undefined`. If `play` doesn't pass `ViewPort` then just remove it from `nextState`